src/metabase/pulse/render/body.clj

```diff
@@ -512,14 +512,51 @@
       [:img {:style (style/style {:display :block :width :100%})
              :src   (:image-src image-bundle)}]]}))
 
+(defn- largest-range
+  [{:keys [fingerprint]}]
+  (let [{mn :min mx :max} (get-in fingerprint [:type :type/Number])]
+    (- mx mn)))
+
+(defn- overlap
+  [col-a col-b]
+  (let [[col-a col-b] (reverse (sort-by largest-range [col-a col-b]))
```

Comment:
is this sort necessary? at first read I don't see why

Reply:
On second thought, you're right, I just don't need it.